.env.example

```diff
@@ -3,4 +3,4 @@ TWITTER_API_SECRET=
 TWITTER_ACCESS_TOKEN_KEY=
 TWITTER_ACCESS_TOKEN_SECRET=
 
-PIC_STORAGE_PATH=/go/src/github.com/ahmedash95/shareAsPic/pics/
\ No newline at end of file
+PIC_STORAGE_PATH=./pics/
```

Comment:
we need the absolute path to use it for photo upload, I can use os.Getwd() but I prefer to have the full path, can be fixed to /app/pics

Reply:
what is the problem in this ?
it will upload to dir pics, in your WORKDIR in Docker Image

Reply:
not needed. ignore my comment :D